index.html

```diff
@@ -1632,7 +1632,6 @@ <h4>
         If the |recordType| value is "`smart-poster`", or if running
         <a>validate external type</a> on |recordType| returns `true`,
         then return the result of running <a>parse records from bytes</a> on |bytes|.
-        Re-[= exception/throw =] any exceptions.
       </li>
       <li>
         Otherwise, [= exception/throw =] a
```

Comment:
There is still a throw here?

Reply:
Instead of re-throwing, should we return `null`?

Reply:
I thought there wouldn't be any anymore as we're always returning either |records| or null